pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -922,11 +851,7 @@ func applyLoadBalancer(c *cluster.Cluster, lb *networking.LoadBalancerSettings,
 	}
 }
 
-func applyLocalityLBSetting(
-	locality *core.Locality,
-	cluster *cluster.Cluster,
-	localityLB *networking.LocalityLoadBalancerSetting,
-) {
+func applyLocalityLBSetting(locality *core.Locality, cluster *cluster.Cluster, localityLB *networking.LocalityLoadBalancerSetting) {
```

Comment:
Weird, shouldn't this return a lint error(lll )

Reply:
sorry did not get. Which lint error?

Reply:
I see. Not sure why there are lines longer than this for example `applyUpstreamTLSSettings`